packages/components/atoms/f-button/src/components/Button.vue

```diff
@@ -67,6 +85,14 @@ export default {
         isLoading: {
             type: Boolean,
             default: false
+        },
+        hasTrailingIcon: {
```

Comment:
I'd condense this into one prop `hasIcon` that can take values `false`, `trailing` or `leading`. That way you never get into a weird spot where people could set true for both of these props and you have to decide which classes to apply.